TESTING.md

```diff
@@ -0,0 +1,59 @@
+# Testing CP-Ansible
+
+Each role has Molecule configuration which enable fast and easy development and testing with Docker containers.
+
+## Prerequisites
+
+Make sure Docker is running and increase the CPU/Memory available to Docker in your Preferences. Next install these python libraries:
+```
+pip install molecule docker
+```
+
+## Using Molecule
+
+Molecule comes with a CLI. Below are instructions for a few of its commands:
+
+### Running a role
+
+Each role has a default scenario in it already so running a role locally is as easy as:
+```
+cd roles/<role-name>
+molecule converge
+```
+
+### SSHing into a container
```

Comment:
Formatting.  I wouldn't make the commands headings, I would just list them out under "Using Molecule".

Reply:
I would also add a link to the molecule docs for these commands: https://molecule.readthedocs.io/en/latest/usage.html